llvm/include/llvm/ADT/APFloat.h

```diff
@@ -306,7 +306,8 @@ struct APFloatBase {
 
 namespace detail {
 
-class IEEEFloat final : public APFloatBase {
+// Not final for testing purposes only.
```

Comment:
I'd prefer to leave it as final

Reply:
`addOrSubtractSignificand` went from private to protected. It is the only non-fishy way to get access for testing.